routes/defaults.php

```diff
@@ -8,12 +9,7 @@
 require __DIR__.'/auth.php';
 
 Route::get('/', function () {
-    return Inertia::render('Welcome', [
```

Comment:
Keep these lines of code in, and change the 'Welcome' to the path and name of the Products component